site/app/views/NavigationView.php

```diff
@@ -14,6 +14,7 @@ public function showGradeables($sections_to_list) {
         $semester = $this->core->getConfig()->getSemester();
         $course = $this->core->getConfig()->getCourse();
         $site_url = $this->core->getConfig()->getSiteUrl();
+        $columnSpan = 10;
```

Comment:
Well I tried doing that and it didn't work so I just got frustrated and hard coded 10 in. I forgot to rid of the variable though.